expression/builtin_time.go

```diff
@@ -141,18 +141,37 @@ var (
 	_ builtinFunc = &builtinUTCTimestampWithoutArgSig{}
 	_ builtinFunc = &builtinExtractSig{}
 	_ builtinFunc = &builtinArithmeticSig{}
-	_ builtinFunc = &builtinUnixTimestampCurrentSig{}
+	_ builtinFunc = &builtinAddDatetimeAndDurationSig{}
+	_ builtinFunc = &builtinAddDatetimeAndStringSig{}
+	_ builtinFunc = &builtinAddTimeDateTimeNullSig{}
+	_ builtinFunc = &builtinAddStringAndDurationSig{}
+	_ builtinFunc = &builtinAddStringAndStringSig{}
+	_ builtinFunc = &builtinAddTimeStringNullSig{}
```

Comment:
Can we merge line 146, line 149 and line 152 into one sig, if the types of the return values are the same?

Reply:
They have different return types. I think it's clearer to seperate them part.